tests/Unit/Repositories/ModularPageRepositoryTest.php

```diff
@@ -679,4 +679,52 @@ public function promo_page_set_columns(): void
 
         $this->assertTrue(!empty($components['catalog-1']['component']['columns']));
     }
+
+    #[Test]
+    public function replace_modular_page_relative_url_with_filename_if_on_base(): void
+    {
+        $page_id = $this->faker->numberbetween(10, 50);
+        $promo_group_id = $this->faker->numberbetween(1, 3);
+
+        // Fake return
+        $return['promotions'] = app(GenericPromo::class)->create(5, false, [
+            'relative_url' => '/promo/image.jpg',
+            'filename_url' => 'https://base.wayne.edu/promo/image.jpg',
```

Comment:
They will never be accessed, only for test purposes to make sure the relative url is replaced. I could change them, but I skipped going through the effort of faking it and making sure the fakes matched because they are only used for the test.

Reply:
Thanks for this explanation, this makes sense. 😄